### PR TITLE
Explain SSA JSON/YAML compatibility bug

### PR DESCRIPTION
**Pull request by pegasas**
resolve: https://github.com/kubernetes/website/issues/45751

### CODE REVIEW

File: content/en/docs/reference/using-api/server-side-apply.md [hunk 1]
```diff
@@ -196,7 +196,8 @@ as [YAML](https://yaml.org/), with the media type `application/apply-patch+yaml`
 Whether you are submitting JSON data or YAML data, use
 `application/apply-patch+yaml` as the `Content-Type` header value.
 
-All JSON documents are valid YAML.
+Most JSON documents are accepted as YAML by Kubernetes. 
+you can find more details on this [open issue](https://github.com/go-yaml/yaml/issues/797)
```
Comment (sftim) [inline content/en/docs/reference/using-api/server-side-apply.md:200]: Let's keep “All JSON documents are valid YAML” as this remains true.

```suggestion
All JSON documents are valid YAML. However, Kubernetes has a bug where it uses a YAML
parser that does not fully implement the YAML specification. Some JSON escapes may
not be recognized.
```
We should link to a bug report in https://github.com/kubernetes/kubernetes; the _Kubernetes_ bug is partly down to our choice of parser library. (If we switched to a different library, the problematic behavior might go away).

**Review by sftim — COMMENTED** [review]
Thanks; however, I recommend a different approach to explaining the bug.

**Comment by sftim** [comment]
@pegasas - we need to sort out a concern here; sorry about that.

I'm not keen on merging this as is. @tengqm have I persuaded you that the comments in my review are worth holding up a merge for?

/hold
to highlight the disagreement here (@tengqm and I are joint technical leads for SIG Docs)

SIG Docs leadership, you can unhold this if either:
- you're confident the disagreement is resolved
- the SIG has reached an overall consensus, but the disagreement didn't get covered

File: content/en/docs/reference/using-api/server-side-apply.md (hunk 1 above)
Comment (pegasas) [reply content/en/docs/reference/using-api/server-side-apply.md]: done.

**Comment by salaxander** [comment]
/lgtm

(adding as PR wrangler shadow)

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**